angular/projects/public-nrpti/src/app/records/administrative-sanctions/administrative-sanction-detail/administrative-sanction-detail.component.ts

```diff
@@ -16,10 +18,14 @@ export class AdministrativeSanctionDetailComponent implements OnInit, OnDestroy
   public loading = true;
   public activeTab = 'detail';
 
+  public entityString = '';
+
```

Comment:
Once you put the function of building the string into a class, you won't need to declare this here anymore.

Reply:
As far as I can tell this is still necessary (and accompanying logic).

I tried to move this (`new Entity(this.data.issuedTo).buildEntityString()`) to the HTML, but angular doesn't like it. 
I did try declaring the class in the TS, so that the HTML would recognize `Entity` but no dice.